microsoft-365/compliance/ome-faq.md

```diff
@@ -160,7 +160,7 @@ Not at this time.
 
 Yes! Encrypted messages are supported for a shared mailbox.
 
-- Users can open protected mails in a shared mailbox where the shared mailbox received a protected mail as part of a distribution group.
+- Users can open protected mails in a shared mailbox as long as they have full access to the mailbox. Please note that "automapping" is also a requirement for the users to be able to open the content in the shared mailbox.
```

Comment:
Editors, change the second line as follows (not great since it doesn't describe automapping, but i'll take it):

Automapping is also required for users to open content in a shared mailbox.